src/collections/_documentation/workflow/issue-owners.md

```diff
@@ -3,57 +3,77 @@ title: 'Issue Owners'
 sidebar_order: 2
 ---
 
-The Issue Owners feature allows you to reduce noise by directing notifications to specific teams or users based on a path or URL. This allows you to get issues into the hands of the developer who can fix them, faster.
+The Issue Owners feature allows you to create rules to decide which user or team should own an [Issue]({%- link _documentation/data-management/event-grouping/index.md -%}). These rules resemble a typical code owners file in a repository, and can match on file paths of files in the stack trace, URL of the request, or event tags. You can automatically assign issues to their owners and alert them about it, allowing you to find the developer with the most context about a fix.
 
 ## How It Works
 
-Issue owners builds upon your alert rules to specify who to notify about a given issue (to learn more about alert rules, click [here](https://blog.sentry.io/2017/10/12/proactive-alert-rules)).
+While the feature is called Issue Owners, Sentry matches ownership rules against individual events in an issue. This has implications for different parts of Sentry that rely on this feature, which we describe in more detail on this page. But this is generally not a problem, and you can think of issue owners as an issue-level property.
```

Comment:
I think that we shouldn't call out bad naming :)  Can we revise to focus on what we do, not on the fact that it's a bit of an oddball name?

"Sentry matches ownership rules against individual events in an issue. This matching is relied upon in other areas of Sentry, described in the following sections. One key item: Issue owners are set for an issue, not a project or organization"

Is that what this para means?

Reply:
It's not oddball naming .. the feature _is_ supposed to be applicable to issues, but the _implementation_ is oddball. So I think we should explicitly call it out as an unexpected thing so people know something funky is going on

Does that make the original sentence make more sense?